website/server/libs/chatReporting/groupChatReporter.js

```diff
@@ -41,10 +41,12 @@ export default class GroupChatReporter extends ChatReporter {
     if (!message) throw new NotFound(this.res.t('messageGroupChatNotFound'));
     if (message.uuid === 'system') throw new BadRequest(this.res.t('messageCannotFlagSystemMessages', {communityManagerEmail: COMMUNITY_MANAGER_EMAIL}));
 
-    return {message, group};
+    const userComment = this.req.body.comment;
+
+    return {message, group, userComment};
   }
 
-  async notify (group, message) {
+  async notify (group, message, userComment) {
```

Comment:
In the `sendTxn` code just below here, please change `flag-report-to-mods` to `flag-report-to-mods-with-comments` (that's a new template I've just made in MailChimp). Also add `REPORTER_COMMENT` to the `emailVariables` with the content being `userComment` or an empty string if the reporter did not make any comment.